src/routes/chart/get.ts

```diff
@@ -19,7 +17,11 @@ interface Response {
   series: {
     pinHashId: string;
     quantityHashId: string;
-    measurements: Measurement[];
+    measurementsSummary: {
```

Comment:
a) What's the reason for including it here instead of one level higher up?
b) What's the reason for including it at all?
c) if we include it, it should have a1b2c3 as example instead of po177.